tests/clar.c

```diff
@@ -124,6 +124,8 @@ static struct {
 	int exit_on_error;
```

Comment:
Putting this at the topmost: the commit message is a bit off, I think. Better would be something like 

```
clar: add option to increase verbosity even 

Here comes the rationale for why you did that change and
what changes because of that. It looks like you could even
copy your PR description for most of that.
```

Reply:
done 👍